tests/benchmark/compute/instruction/test_account_query.py

```diff
@@ -371,27 +372,47 @@ def test_ext_account_query_warm(
     benchmark_test: BenchmarkTestFiller,
     pre: Alloc,
     opcode: Op,
-    absent_target: bool,
+    empty_account: bool,
+    initial_balance: bool,
+    initial_storage: bool,
 ) -> None:
     """
     Test running a block with as many stateful opcodes doing warm access
     for an account.
     """
     # Setup
-    target_addr = pre.empty_account()
+    target_addr = Address()
+    if not initial_balance and not initial_storage:
+        target_addr = pre.empty_account()
+    elif initial_balance or initial_storage:
```

Comment:
I think we should refactor this logic a bit:
```suggestion
    if not initial_balance and not initial_storage and empty_code:
        target_addr = pre.empty_account()
    else:
```
And in the `else` branch we construct the `kwargs` to either call `pre.fund_eoa` or `pre.deploy_contract`.